config/config.yaml

```diff
@@ -148,6 +148,7 @@ policy:
               features:
                 - el7
                 - plugin-compiler-fix-vendor
+                - golang-x-bullseye-suffix
```

Comment:
Can this conditional be eliminated? The golang cross variable for the branch can be directly used, can it not?

Reply:
When the matrix job is invloved, it becomes a problem. For release-5-lts, the job has 1.16-bullseye and 1.16-el7 as the base golang cross images(https://github.com/TykTechnologies/tyk/blob/32f8b82fb4fa176bf9a68cd841c3039593081e55/.github/workflows/release.yml#L35) Without renaming/retagging the golang-cross images - this seemed like the only way we could keep it in templates.

Reply:
Perhaps retagging golang-cross images is a better solution?

Reply:
Retagging 1.16-bullseye to 1.16 should solve this. I'll try and check if `1.16` image is used anywhere else, and if it could be replaced by `1.16-bullseye`

Reply:
Hmmm, perhaps you could explain what the problem is? Adding features is the last resort from the long term maintenance perspective.

Reply:
Te problem is what I explained above - the non-el7 and el7 matrix job run can't be run without a conditional suffix right now